checked/bytes.go

```diff
@@ -54,6 +54,7 @@ func NewBytes(value []byte, opts BytesOptions) Bytes {
 		value: value,
 	}
 	b.SetFinalizer(b)
+	b.TrackObject(b.value)
```

Comment:
Ah I see you do this with an atomic load.  Perhaps just a bool to avoid slightly slower atomic load?  We really should only enable/disable this at the very start of runtime.  That way we can avoid data races by only using the setters at runtime startup.